### ISSUE TITLE
In the tutorial 5 of snnTorch the data that are inserted for training are the classic mnist and not spikes

### ISSUE THREAD

**Issue by TheoTsio** [jeshraghian/snntorch#262]
* snntorch version:
* Python version:
* Operating System:

### Description

train_loader = DataLoader(mnist_train, batch_size=batch_size, shuffle=True, drop_last=True)
test_loader = DataLoader(mnist_test, batch_size=batch_size, shuffle=True, drop_last=True)

It is using just this as input and there are no spikes

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by jeshraghian** [comment]
The inputs don't have to be spikes. The idea is that the network will learn the optimal representation to convert non-spiking data into a format compatible with spiking. Tutorial 2 shows how non-spiking data is injected as a time-varying input current. The same logic applies.
